src/benchmarks/micro/coreclr/BenchmarksGame/regex-redux-5.cs

```diff
@@ -26,78 +25,57 @@ namespace BenchmarksGame
     [BenchmarkCategory(Categories.CoreCLR, Categories.BenchmarksGame)]
     public class RegexRedux_5
```

Comment:
Not specifically related to this change, but how are we thinking about these benchmark categories in light of dotnet/runtime?  It was already a little strange having this benchmark categorized as "coreclr" when it's more about Regex, and now that we have dotnet/runtime instead of dotnet/coreclr + dotnet/corefx, the distinction seems even more odd.

Reply:
I agree that it seems odd now. However, I think that Runtime devs are still mostly interested in running the "CoreCLR" benchmarks only. I am going to wait to hear more feedback. Most probably we are going to remove it within the next few months.

Reply:
Yeah. The categories really represent "what repo the test used to live in."

I think we should ditch the CoreCLR/CoreFX distinction and come up with categories that are meaningful to the test. There's no reason we can't have "Serializers", "Regex", etc, etc.

I also think that whatever we put in the attribute should be a specific descriptor of the test and what functionality area it targets, and nothing about grouping or organization. That way when we inevitably re-organize again in three years we don't have to go touch a ton of places.

For example, if we want "metacategories" like CoreCLR so that folks can run groups of tests, I think we should build a feature into either our argument parser or BDN directly to allow grouping those. We change the categories on a given test to better match what it does, and then if someone specifies a metacategory we transform it to everything it maps to.